### PR TITLE
Update how-to-sign-in-with-metamask-python-django.md

### PR DESCRIPTION
**Pull request by imperorrp**
Tutorial edits as a companion to the associated pull request proposed [here](https://github.com/MoralisWeb3/demo-apps/pull/71)

### CODE REVIEW

**Review by imperorrp — COMMENTED** [review]
The current moralis django authentication system doesn't work due to the ExpirationTime having long expired- this fixes it without assigning an arbitrary expiration date but assigning a reasonable amount of time for the Moralis auth api request to be processed.

**Comment by YosephKS** [comment]
> The current moralis django authentication system doesn't work due to the ExpirationTime having long expired- this fixes it without assigning an arbitrary expiration date but assigning a reasonable amount of time for the Moralis auth api request to be processed.

Looks good! @imperorrp test it out on my local as well and thanks for this, we might have missed it~

One thing before merging will appreciate if you can delete the `print` line 😄 thanks!

File: docs/03-authentication-api/tutorials/how-to-sign-in-with-metamask-python-django.md [hunk 1]
```diff
@@ -126,14 +127,21 @@ def request_message(request):
     data = json.loads(request.body)
     print(data)
 
+    #setting request expiration time to 1 minute after the present->
+    present = datetime.now(timezone.utc)
+    present_plus_one_m = present + timedelta(minutes=1)
+    expirationTime = str(present_plus_one_m.isoformat())
+    expirationTime = str(expirationTime[:-6]) + 'Z'
+    print(f'ExpirationTime = {expirationTime}')
```
Comment (YosephKS) [inline docs/03-authentication-api/tutorials/how-to-sign-in-with-metamask-python-django.md:135]: Just need to remove this and you are good to go~

**Review by YosephKS — APPROVED** [review]
One thing here to remove

File: docs/03-authentication-api/tutorials/how-to-sign-in-with-metamask-python-django.md (hunk 1 above)
Comment (imperorrp) [reply docs/03-authentication-api/tutorials/how-to-sign-in-with-metamask-python-django.md]: Sure, removed it

**Review by YosephKS — APPROVED** [review]

**Pull request merged**